src/features/click-to-load.js

```diff
@@ -662,6 +669,7 @@ function createPlaceholderElementAndReplace (widget, trackingElement) {
 
     // Facebook
     if (widget.replaceSettings.type === 'dialog') {
+        ctl.addDebugFlag()
```

Comment:
I consulted with @kzar and we agreed to make this flag just match the existing breakage flags.

Reply:
The intent is to drop the other one eventually right?

Reply:
yes, I expect to convert the existing breakage flags to the generic format